utils/vertical_slice/mount_agnesi.py

```diff
@@ -0,0 +1,149 @@
+import firedrake as fd
+from utils.compressible_flow import hydrostatic_rho
+from utils.misc import function_maximum
+
+
+def L(hydrostatic=False):
```

Comment:
consider doing a class that stores hydrostatic and then making these cached properties